### ISSUE TITLE
Add more languages

### ISSUE THREAD

**Issue by b-lam** [b-lam/Resplash#14]
Add all strings in app to strings.xml and get people to translate it.

**Comment by GameRebellStudio** [comment]
I can help with Czech translation :-)

**Comment by b-lam** [comment]
@GameRebellStudio Essentially all you need to do is translate the string in this file 
[
](https://github.com/b-lam/Resplash/blob/develop/app/src/main/res/values/strings.xml). Feel free to skip any of the string that you don't think need translating or are too difficult or don't have a good translation. Thanks. Let me know if you have any questions!

**Comment by GameRebellStudio** [comment]
@b-lam I can link you translated file in few days (I'm on a vacation now) in Google Docs or should I use another way? I'll try to do my best as fast as possible ;-)

**Comment by b-lam** [comment]
Google drive might work a bit better but anything works for me. And no rush, enjoy your vacation!

**Comment by GameRebellStudio** [comment]
Hello!
It took me more time than I expected but it's done. I tried my best so there shouldn't be any typing errors. In some cases I wasn't sure what exactly should I use _(sometimes there are more than 5 Czech equivalents for 1 English word)_ but I always chose thing that fits and sounds great in context. There were some issues with transforming few English words we assumed into colloquial Czech but aren't used in writing Czech. We also have a bit more complex declension and conjugation.
Nevertheless everything is ready and checked twice (maybe five times :D).
Here is link to Google Drive: [https://docs.google.com/document/d/10tMVY8bHf3EWaTxlD2tPpQ6iERnmJMIRUKcCD1BM7SI/edit?usp=sharing]

Best regards,
Game Rebell Studio (GmRbL Labs)

**Comment by b-lam** [comment]
@GameRebellStudio Thanks man! Looks good. I'll try and add them into the app as soon as I get some free time :)

**Comment by GameRebellStudio** [comment]
@b-lam
I'm glad I can help :-) Don't hurry, enjoy your free time.

**Comment by svetaz** [comment]
I can help you with Serbian translation if you still need it :)

**Comment by b-lam** [comment]
@svetaz Yeah! That would be great! Translations are always welcome :) All you need to do is translate the strings in this file [https://github.com/b-lam/Resplash/blob/master/app/src/main/res/values/strings.xml](https://github.com/b-lam/Resplash/blob/master/app/src/main/res/values/strings.xml) Feel free to skip any string that you don't think needs translating or doesn't have a good translation. Let me know if you have any questions.

**Comment by svetaz** [comment]
No problem,give me a day or two and it will be done,this app's aesthetics and functionality blew my mind, it's the least I can do :)

**Comment by svetaz** [comment]
Translation done,its on: [](https://app.box.com/s/0zaadttysni0jedzp4jbtkfuibry342e)
I will check it again once you push the translation live so we can make fast changes before anyone notices :)  

Cheers ! :+1:

**Comment by b-lam** [comment]
@svetaz Done! Update should be available now.

**Comment by svetaz** [comment]
@b-lam thanks, i will check the app and add some corrections if needed :)

**Comment by KrazyCavin** [comment]
I just made a zh-cn (Simplified Chinese) translation, hope you will like it ;)
https://gist.github.com/KrazyCavin/2d8602bf688082e110117e3959d61706

**Comment by b-lam** [comment]
@KrazyCavin Thanks for the translations! I have exams for the next few weeks so I probably won't have time to add them in soon. I'll add them in the new year as well as the bug that you reported and a bunch of other stuff. Thanks again :)

**Comment by enriquejro** [comment]
@b-lam I've just made an spanish (es_ES) translation. Thank you for your awesome app! 
[spanish es_ES.zip](https://github.com/b-lam/Resplash/files/3280344/spanish.es_ES.zip)

**Comment by b-lam** [comment]
@enriquejro Awesome! I'm sure this will make a lot of people happy 😀. I'll add the translations in once I get some free time

**Comment by Lead82** [comment]
@b-lam Italian (it-IT) translation here, i hope it's appreciated :wink:.
Keep up the good work man!
[https://drive.google.com/file/d/1tSqpkYkFGBFlnO0x27woQqyD_m3j2qnS/view?usp=sharing](url)

**Comment by b-lam** [comment]
@Lead82 Thanks so much for the translation! I'm working on an update right now so I'll be sure to add this in asap.

**Comment by A1pen** [comment]
Hi, 
I've added a german translation. Pull request is available here: https://github.com/b-lam/Resplash/pull/76

THX for developing the app, really love it!

**Comment by b-lam** [comment]
@A1pen Thanks for the translation! I'll take a look at the PR now :)

**Comment by boran2000** [comment]
I have some great news.😀 I just finished the Turkish translation and i am doing last controls in case of some typing errors. I will upload the translation in just couple minutes. I hope it's appreciated.

**Comment by boran2000** [comment]
https://drive.google.com/file/d/1N7fsJLe70o73P0P55wqg3znz0XYDUM_n/view?usp=sharing  for Turkish (tr-TR) translation.

**Comment by b-lam** [comment]
@boran2000 Amazing! I'm adding them in right now. I'll release an update in a few days with your translations :)

**Comment by almaftuchin** [comment]
Manual translation for Indonesia Language (Bahasa Indonesia) here: [Indonesia (ID Strings).xml](https://drive.google.com/file/d/1Wg6FwgaFuKvksZUSw7vTRCIS8OXaFD3w/view?usp=sharing)

**Comment by alistairholmes** [comment]
Hi there :wave: first of all huge thanks for the inspiration, your app helped me to add some cool features to my apps. I especially liked the about screen. Unfortunately I can't buy you a coffee, but I can contribute :wink: Can I add a French 🇫🇷  translation for your app?

**Comment by b-lam** [comment]
That would be great! Translations help out a lot:) All you need to do is translate the strings in this file https://github.com/b-lam/Resplash/blob/master/app/src/main/res/values/strings.xml It would be great if you could keep it in the XML format. Let me know if you have any questions.

**Comment by AlumiK** [comment]
Hi, Here's a new simplified Chinese translation: [GitHub Gist link](https://gist.github.com/AlumiK/b32076187f888a744504528192137499).

**Comment by b-lam** [comment]
@AlumiK Thanks! I assumed that Google translate would have failed me on some of the translations lol

**Comment by Kenta1561** [comment]
Hi, first time contributor here. I'd like to add Japanese translation as well as update the German translation because some of the translations don't sound natural. I'll open a pull request as soon as I have made some changes.

**Comment by b-lam** [comment]
@Kenta1561 Awesome! Japanese translations would be great! 😃 I think most of the language translations are going to be a bit rough since I had to use Google translate for a bunch of new strings. So any fixes are greatly appreciated.
